picasso/src/test/java/com/squareup/picasso3/NetworkRequestHandlerTest.java

```diff
@@ -203,6 +204,26 @@
     assertThat(latch.await(10, SECONDS)).isTrue();
   }
 
+  @Test public void shouldHandleSchemeInsensitiveCase() {
+    String[] schemes = {
+            "http",
+            "https",
+            "HTTP",
+            "HTTPS",
+            "HTtP",
+    };
+
+    for (String scheme : schemes) {
+      final Uri uri = URI_1.buildUpon().scheme(scheme).build();
+      final Boolean shouldHandle = networkHandler.canHandleRequest(
+              TestUtils.mockRequest(uri)
```

Comment:
i'd inline the `mockRequest` for tests, but i'm hardly opinionated here.

Reply:
I didn't inline because i would need to import the whole package there, from picasso instead of okhttp.. :/